Jules.md

```diff
@@ -0,0 +1,108 @@
+# Jules Onboarding Guide for `firebase-android-sdk`
+
+This guide provides essential information for working within the `firebase-android-sdk` repository.
+
+## Table of Contents
+- [Environment Setup](#environment-setup)
```

Comment:
Do these links work?

Reply:
Yes, at least in Github's view https://github.com/firebase/firebase-android-sdk/blob/update-jules-md/Jules.md#testing